client/blocks/inline-help/popover.jsx

```diff
@@ -97,6 +112,12 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	switchToClassicEditor = () => {
+		const { siteId, optOut, classicUrl } = this.props;
+		optOut( siteId );
+		page.redirect( classicUrl );
```

Comment:
We should avoid using `page` directly in our components, and dispatching a [`navigate` action](https://github.com/Automattic/wp-calypso/blob/master/client/state/ui/actions.js#L89-L95) instead.